introspection/parse.go

```diff
@@ -284,7 +286,7 @@ func (p parser) buildInputValue(input *InputValue) *ast.ArgumentDefinition {
 	if input.DefaultValue != nil {
 		defaultValue = &ast.Value{
 			Raw:      pointerString(input.DefaultValue),
-			Kind:     ast.Variable,
+			Kind:     p.parseValueKind(typ),
```

Comment:
ast.Variable for query value. following $id is ast.Variable.

```graphql
query Query($id: ID!) {
    node(id: $id) {
        id
    }
}
```

Argument Default Value is ast.StringValue, ast.IntValue and so on.